__test__/List.test.js

```diff
@@ -0,0 +1,63 @@
+import List from '../src/modules/List.js';
+// import localStorageMock from './__mock__/localStorage.js';
+
+const list = new List();
+// global.localStorage = localStorageMock();
```

Comment:
- [ ] please try to remove the unused commented code in order to keep the file clean and professional